Gems/ROS2/Code/Source/RobotImporter/Utils/SourceAssetsStorage.cpp

```diff
@@ -535,10 +560,48 @@ namespace ROS2::Utils
         return true;
     }
 
-    bool CreateSceneManifest(const AZ::IO::Path& sourceAssetPath, bool collider, bool visual)
+    bool CreateSceneManifest(const AZ::IO::Path& sourceAssetPath, const bool collider, const bool visual)
     {
-        auto assetInfoFilePath = AZ::IO::Path{ sourceAssetPath };
-        assetInfoFilePath.Native() += ".assetinfo";
         return CreateSceneManifest(sourceAssetPath, sourceAssetPath.Native() + ".assetinfo", collider, visual);
     }
+
+    AZStd::unordered_set<AZ::IO::Path> GetMeshAssets(const AZ::IO::Path& sourceAssetPath)
```

Comment:
Suggestion: Maybe `GetMeshTextureAssets` instead of `GetMeshAssets`. To me, `GetMeshAssets` sounds like it's returning meshes, not textures.

Also, maybe `sourceMeshAssetPath` instead of `sourceAssetPath` to make it clear that it's expecting the mesh file as input.